### DISCUSSION TITLE
Core WCF in Linux

### DISCUSSION THREAD

**Discussion by vvijayalakshmi21** [CoreWCF/CoreWCF#324] [General]
Hi,

I understand that CoreWCF can help migrate servers from .net framework to .net core/.net 5 but I am not sure if it can run on all platforms including Linux. Please can you help clarify?

If it cannot run on non windows platform, please can you suggest any alternatives available?

**Comment by birojnayak** [comment]
hi @vvijayalakshmi21 it works in Linux . We have tests running both in Windows and Linux platform. The objective of this project is to ensure that it works in cross platform.  Features which are available as of latest release, can be referred here https://github.com/CoreWCF/CoreWCF/releases/tag/v0.1.0 and features to be prioritized is here https://github.com/CoreWCF/CoreWCF/issues/234 .

Biroj.
